pkgs/servers/web-apps/ethercalc/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv
+, pkgs
+, lib
+, nodejs-14_x
+}:
+
+let
+  nodejs = nodejs-14_x;
+
+  nodePackages = import ./node-packages.nix {
+    inherit pkgs nodejs;
+    inherit (stdenv.hostPlatform) system;
+  };
+
+  ethercalc = lib.head (lib.attrValues nodePackages);
+
+  combined = ethercalc.override {
+    meta = with lib; {
+      description = "Online collaborative spreadsheet";
+      license = licenses.free;
```

Comment:
looks like ethercalc is CPAL 1.0, there's `cpal10` for that :)

Reply:
Well, some parts of it are CPAL 1.0, but the repository README indicates that some source files are under lots of different licenses. I wasn't sure what the value is to list all of those here. What do you recommend?

Reply:
if multiple licenses apply you can also put a list here with all of them instead of just a single license.

Reply:
Okay, will do so in incoming force push :-) (I'm just slightly worried that people might misunderstand the list of licenses as "I have the option" (like with many GPLv2+ licensed software, where I as a user can choose which version to use) instead of "all of those licenses apply, each to different files".)

Reply:
that's a legitimate worry. :thinking: might be worth asking about on discourse and amending the manual with the results?

Reply:
Did so in force push. (It's just that I'm feeling a little bit uneasy: Does a list of licenses mean that I, as a user, can choose which license to pick (as with many GPLv2+ software)? Or does it mean that different source files are governed by different licenses?)

Reply:
Will do so! (though probably not today)